### ISSUE TITLE
Primary navigation component has low text contrast

### ISSUE THREAD

**Issue by Helsinki-Vice** [wvuweb/wvu-design-system-v2#109]
# Problem
Color contrast on the Primary Navigation component is too low and is hard to read. Although it _technically_ passes WCAG 2.1 AA with a ratio of 4.8, it fails on the much more accurate APCA standard. The current contrast algorithm is somewhat flawed and will be dropped by the W3C with the introduction of WCAG 3 in the coming year or two. Even without running the numbers, it's pretty evident that a 16px 400 weight font in `rgba(255, 255, 255, 0.5)` should not be displayed on a `rgb(28, 43, 57)` background.
`
# Steps to reproduce the issue

1. [Go to the Primary Navigation component](https://designsystem.wvu.edu/components/essential-elements/nav).
2. Visit any number of [color contrast checkers on the web](https://apcacontrast.com/?BG=1c2b39&TXT=ffffff80&DEV=G4g&BUF=A22).

# Results 
 
Observe failing contrast of Lc -40.9.

# How to fix

1. Edit the selector `.wvu-site-nav__items .nav-link`
2. change `color` to from  `rgba(255, 255, 255, 0.5)` to `#FFF`
